README.md

```diff
@@ -20,13 +20,13 @@ It includes an optional label,error message,invalid,autovalidate and required at
 <hipaa-dea-number-input required="boolean value"></hipaa-dea-number-input>
 ```
 ### DEA Numbers Validation.
-A DEA number (DEA Registration Number) is a number assigned to a health care provider.
+#### A DEA number (DEA Registration Number) is a number assigned to a health care provider.
 
-A valid DEA number consists of: 2 letters, 6 numbers, & 1 check digit.
-The first letter is a code identifying the type of registrant.
-The second letter is the first letter of the registrant's last name.
-
-The last Number : DEA numbers can be verified by using the last number, which is known as the Check Digit.
+A valid DEA number consists of total 9 characters.
+- The first character is a letter, a code identifying the type of registrant. 
```

Comment:
@cherukumilli 
The documentation in the code is from following wikipedia article
`https://en.wikipedia.org/wiki/DEA_number`